pycascades/pycascades-cms
# Add a Venue page template

This PR adds a Venue page template a la https://2020.pycascades.com/venue/. Most of the structure & JS was pulled directly from that page, so if there are improvements to be made on it please point them out :slightly_smiling_face:.

This includes some CSS changes, particularly to the image attribution section. Previously, even though we were allowing HTML, none of the anchor tags were actually clickable. This now ensures that the attribution links can be clicked. It's a little finicky on the home page still because the navbar appears over the image, but that's fine IMO.

I also fixed an issue where one of our divs was never being closed! And I changed the `extra_js` override to not actually clobber some other, apparently essential, javascript.

https://user-images.githubusercontent.com/10214785/209873747-45b95357-c466-4e7b-8219-4b4e51819685.mp4


## Testing instructions

1. `just manage migrate`
1. Add a new page
1. Select the "Venue page" template
1. Switch to promote and check "Show in menus"
1. Add some data to the fields (see below for example data)
1. View the page and make sure it looks like the video above

### Sample data:

- Venue content:
  ## SFU Harbour Centre
  Here's a [link to the site](https://www.sfu.ca/mecs/vancouver-main/harbour-centre/meeting-rooms/joseph---rosalie-segal-centre.html).

- Venue image attribution:
  SFU Website

- Location address:
  **SFU Harbour Center**
  555 W Hastings St
  Vancouver, BC V6B 4N6, Canada

- Location lat: 49.28480187829870000
- Location long: -123.11175894603200000
- Location google maps url: https://goo.gl/maps/BPuKFzhbiWK4osvW8
